bot/cogs/polling.py

```diff
@@ -34,32 +34,33 @@ async def on_ready(self):
 
     async def new_poll(self, pollings, server: str, poll: str):
         while True:
-            #try:
+            # try:
             if pollings.get("type") == "player":
                 interval = float(pollings.get("polling_interval")) * 60
                 await asyncio.sleep(interval)
                 logging.info(f"Executing Task {poll} on server {server}")
                 try:
                     state, ctx = await self.player_polling(ctx, pollings, server, state)
                 except:
-                    state, ctx = await self.player_polling("noctx", pollings, server, "none")
+                    state, ctx = await self.player_polling(pollings, server, None, None)
                 # if pollings.get("type") == "autobalance":
                 #    interval = float(pollings.get("polling_interval")) * 60
                 #    await asyncio.sleep(interval)
                 #    logging.info(f"Executing Task {poll} on server {server}")
                 #    await self.autobalance_polling(pollings, server, poll)
-            #except Exception as e:
+            # except Exception as e:
             #    await asyncio.sleep(1)
             #    logging.info(f"Exception appeared in {poll} on server {server}! Exception: {e}")
             #    pass
 
-    async def player_polling(self, ctx, pollings, server, old_state):
+    async def player_polling(self, pollings, server, old_state=None, ctx=None):
         channel = self.bot.get_channel(int(pollings.get("polling_channel")))
         logging.info(f"Starting poll with state: {old_state}")
-        data, ctx = await exec_server_command(ctx, server, "RefreshList", "True")
+        data, ctx = await exec_server_command(ctx, server, "RefreshList", True)
         amt = len(data.get("PlayerList"))
+        p_role = "<@&" + str(pollings.get("polling_role")) + ">"
         logging.info(f"{server} has {amt} players")
-        if pollings.get("show_scoreboard") == "True":
+        if pollings.get("show_scoreboard") == True:
```

Comment:
Not as big, but if you check whether something is `True` or `False` you usually just check
`if pollings.get("show_scoreboard"):`
and for `False`
`if not pollings.get("show_scoreboard"):`